src/cheat_gui.cpp

```diff
@@ -115,6 +115,7 @@ static int32 ClickChangeDateCheat(int32 p1, int32 p2)
 	EnginesMonthlyLoop();
 	SetWindowDirty(WC_STATUS_BAR, 0);
 	InvalidateWindowClassesData(WC_BUILD_STATION, 0);
+	InvalidateWindowClassesData(WC_BUS_STATION, 0);
```

Comment:
Shouldn't the truck stop window also be invalidated?

```suggestion
	InvalidateWindowClassesData(WC_BUS_STATION, 0);
	InvalidateWindowClassesData(WC_TRUCK_STATION, 0);
```